components/feature/pwa/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     kapt Dependencies.androidx_room_compiler
 
     testImplementation project(':support-test')
+    testImplementation project(':support-test-libstate')
 
```

Comment:
We should be able remote `browser-session` here now?

Reply:
Yeah! I created all PRs in parallel and in none I could remove it on its own. But this is now the last one and I can do that here. Will update the PR. :)

Reply:
Argh. Looks like there's an intent processor left that needs refactoring.